src/components/ProfilePage/Leaderboard/Leaderboard.tsx

```diff
@@ -135,21 +208,33 @@ export const Leaderboard = ({ address }: { address?: string }) => {
               pointerEvents: 'none',
             }}
           >
-            <ChevronLeft
-              onClick={previousPage}
-              sx={{ cursor: 'pointer', pointerEvents: 'auto' }}
-            />
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
To argue :) A Box with only two styling props may not need to be a styled-component. But we are using it twice, so maybe it could qualify to become one?